examples/smpi/replay_multiple_manual_deploy/replay_multiple_manual.cpp

```diff
@@ -0,0 +1,337 @@
+/* Copyright (c) 2009-2016. The SimGrid Team.
```

Comment:
Could you please explain the big lines of this file in comments? What it is, what it's good for, how it achieves it (in, say, 5 lines).
Thx in advance.

(plus: please update the copyright year)

Reply:
Done in 3fa91a55d and d253d731a.